lib/flippant/adapters/redis.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "redis"
+require "forwardable"
```

Comment:
Any idea why this needed to be required?

Reply:
Trying to run rspec, it couldn't find forwardable. When I looked it up, it seemed like most places required it. It fixed the issue for me.